README.md

```diff
@@ -152,6 +152,17 @@ To push without running the pre-push hook (e.g. doc updates):
 git push <remote> --no-verify
 ```
 
+Note: If while pushing you encounter this error "Could not initialize class org.codehaus.groovy.runtime.InvokerHelper" then downgrade your java version to java11
+
+Steps to downgrade Java Version: 
```

Comment:
I would make this title more specific 'Steps to downgrade Java Version to 11 on Mac with brew' because there are alternate ways of doing this as well, including downloading from the Java site, and I'm not sure this is verified for Windows.